wrap-send-video-note.go

```diff
@@ -0,0 +1,103 @@
+package tgwrap
+
+import (
+	"fmt"
+	"reflect"
+	"tgwrap/internal/thestruct"
```

Comment:
Please change import to "github.com/rogozhka/tgwrap/internal/thestruct" so the tests pass.

Reply:
Done